hack/update-bazel.sh

```diff
@@ -22,5 +22,8 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 git config http.https://gopkg.in.followRedirects true
 
-go get -u gopkg.in/mikedanese/gazel.v13/gazel
+# Remove generated files prior to running gazel.
+rm -f "${KUBE_ROOT}/pkg/generated/openapi/zz_generated.openapi.go"
```

Comment:
Eventually this will be `make clean_generated` instead, but for now this will have to do.

`gazel` isn't smart enough to do it any other way iiuc.